distribution/build.gradle

```diff
@@ -215,7 +215,7 @@ void copyLog4jProperties(TaskProvider buildTask, Project module) {
 }
 
 ext.restTestExpansions = [
-  'expected.modules.count': 0,
+  'expected.modules.count': "0", //TODO: is it safe to make this a string ?
```

Comment:
This todo might be worth a quick follow u on it's usage to ensure we don't break something with distribution.

Reply:
19762bb